homeassistant/components/http.py

```diff
@@ -194,12 +239,17 @@ class RequestHandler(SimpleHTTPRequestHandler):
     We extend from SimpleHTTPRequestHandler instead of Base so we
     can use the guess content type methods.
     """
-
     server_version = "HomeAssistant/1.0"
 
+    def __init__(self, req, client_addr, server):
+        SimpleHTTPRequestHandler.__init__(self, req, client_addr, server)
+        self._session = None
+        self.cookie = None
+
     def _handle_request(self, method):  # pylint: disable=too-many-branches
         """ Does some common checks and calls appropriate method. """
         url = urlparse(self.path)
+        self._session = None
```

Comment:
I don't think that RequestHandlers are recycled making this statement not have any effect.